server/src/main/java/com/coincoinche/events/CardPlayedEvent.java

```diff
@@ -4,19 +4,25 @@
 
 public class CardPlayedEvent extends Event {
   private String card;
+  private int playerIndex;
```

Comment:
Add the playerIndex to this event so that the front knows who is the player who played this card and can add it to the trick without needing additional information from another event.